tests/e2e/support/objects/app-files/link/actions.ts

```diff
@@ -161,7 +161,7 @@ export const changeName = async (args: changeNameArgs): Promise<string> => {
   await page.locator(editPublicLinkRenameConfirm).click()
   const message = await page.locator(linkUpdateDialog).textContent()
   expect(message.trim()).toBe('Link was updated successfully')
-  return await page.locator(getMostRecentLink + '//h4').textContent()
+  return await page.locator(getMostRecentLink + '//h4[1]').textContent()
```

Comment:
how about getting link by name? because we already have renamed the link in the test scenario